bot/src/auctions.ts

```diff
@@ -1,6 +1,8 @@
 import type { AuctionInitialInfo } from 'auctions-core/src/types';
```

Comment:
Since we introduce another type of the auctions, we should do a bit or refactoring. As proposed, can you please rename this file to `collateral` or even to `auctions/collateral`, moving surplus to the same folder?